cash/cash.c

```diff
@@ -0,0 +1,41 @@
+#include <cs50.h>
+#include <stdio.h>
+
+int main(void)
```

Comment:
there is no any exlanation for this PR so I have no idea what this app is doing. Please attach enough information for reviewer to make it possible to review

Reply:
is it better to add information via comments in the code?

Reply:
it depends. Some of information can be added here in PR, some of it can be written within source code. You can at least attach link to a task, but also brief explanation of the task itself would work